spec/dummy/db/schema.rb

```diff
@@ -37,6 +38,7 @@
     t.string   "name"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.integer  "title"
```

Comment:
A `title` for a user resource? Does it make sense from a business point of view?